website/blog/2026/02/02-18-enhanced-security-for-helm-deployments-gardener-adds-custom-ca-support-for-oci-registries.md

```
@@ -0,0 +1,115 @@
+publishdate: 2026-02-18
+authors:
+- avatar: https://avatars.githubusercontent.com/shafeeqes
+  email: shafeeque.e.s@sap.com
+  login: shafeeqes
+  name: Shafeeque E S
+aliases: ["/blog/2026/02/18/enhanced-security-for-helm-deployments-gardener-adds-custom-ca-support-for-oci-registries"]
+---
+
+Gardener continues to enhance its security and flexibility, particularly for users operating in air-gapped environments or utilizing private infrastructure. A new feature now allows operators to specify a custom Certificate Authority (CA) bundle when pulling Helm charts from OCI registries. This is a significant improvement for environments where registries are secured with custom or self-signed TLS certificates.
+
+### The Challenge of Private Registries
```

markdownlint-cli2 (0.21.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)